### DISCUSSION TITLE
Suggestion: Display the author of a form

### DISCUSSION THREAD

**Discussion by darylknight** [solspace/craft-freeform#481] [Feature Requests]
Filing 3 separate suggestions following feedback from a client. We have a large multi-site Craft install with 8 sites. As the website continues to grow, our client is noticing some growing pains related to Freeform.

Suggestion 2: Please can we see which user created a form and when? In this case, it would be useful to be able to ask that person in the organization whether the form is being used anymore. It's impossible for global leadership to keep track of 100+ pages on each of the 8 multi-sites.

**Comment by kjmartens** [comment]
This (along with some other details) will be available in the next major version of Freeform 🙂

**Comment by kjmartens** [comment]
This has been implemented in Freeform 5 (now in beta). 🙂
